docs/design/design-reviews/recipes/preferred-diagram-tooling.md

```diff
@@ -1,15 +1,30 @@
 # Preferred Diagram Tooling
 
 At each stage in the engagement process, diagrams are a key part of the design review.
-The preferred tooling for creating and maintaining diagrams is the `.drawio` format from [diagrams.net](http://diagrams.net) (formerly [draw.io](http://draw.io))
+The preferred tooling for creating and maintaining diagrams is to choose **one** of the following:
 
-The `.drawio` format uses the metadata layer within the PNG file-format to hide SVG vector graphics representation, then renders the .png when saving.
-This clever use of both the meta layer and image layer allows anyone to further edit the PNG file.
-It also renders like a normal PNG in browsers and other viewers, making it easy to transfer and embed.
-Furthermore, it can be edited within VSCode very easily using the [Draw.io Integration VSCode Extension](https://marketplace.visualstudio.com/items?itemName=hediet.vscode-drawio).
+- Microsoft Visio
+- Microsoft PowerPoint
+- The `.drawio.png` (or `.drawio`) format from [diagrams.net](http://diagrams.net) (formerly [draw.io](http://draw.io))
+
+In all cases, we recommend storing the exported PNG images from these diagrams in the repo along with the source files so they can easily be referenced in documentation and more easily reviewed during PRs. The `.drawio.png` format stores both at once.
+
+## Microsoft Visio
+
+This is the Microsoft diagram tool of choice, the desktop app exists on PC, and there's a great Web app. This is the preferred format of the Azure Architecture Center.
```

Comment:
I think we should hold off on giving a tool of choice, they all have their uses, so I would tone this down a bit, but we can still mention that it is the preferred format for the Azure Architecture Center if they have explicitly expressed that.

Reply:
Thanks for the heads up @TessFerrandez. Done.